### ISSUE TITLE
Time statistic seems to be wrongly calculated

### ISSUE THREAD

**Issue by damianavila** [fabiospampinato/vscode-todo-plus#341]
If I try this one...

```
Baz:
    ✔ Foo @started(21-11-26 10:12) @done(21-11-26 11:17) @lasted(1h5m21s)
    ✔ Bar @started(21-11-26 11:17) @done(21-11-26 11:18) @lasted(1m54s)
```

I see the following:

![Screen Shot 2021-11-26 at 11 21 02](https://user-images.githubusercontent.com/1640669/143594768-8923eae6-c1f5-41d8-9843-333e930e21ae.png)

As you can see, it seems it is actually adding seconds to minutes and doing some miscalculation. Even more, it happens with just one item:

![Screen Shot 2021-11-26 at 11 25 38](https://user-images.githubusercontent.com/1640669/143595286-962ebc67-c0b9-475f-abb7-4be0c78db516.png)

Thoughts?

Btw, latest version and config options I am using now:
```
    "todo.archive.remove.emptyProjects": false,
    "todo.statistics.project.enabled": "global.projects < 100",
    "todo.statistics.project.text": "([pending]) [est] [lasted]",
````

Thanks!!

**Comment by greg-double** [comment]
Yes, I just found this as well. I'm using v4.18.4 and it seems like when there are minutes after the hours and the minutes are less than 10, it drops the minutes in the summary. However, if you inject a zero before the minute, it calculates correctly...

![image](https://user-images.githubusercontent.com/55329854/144493070-a1da2a41-89a2-490c-b23f-3bc8435f0c95.png)

**Comment by Sn0bli** [comment]
in src/utils/time.ts:127 the result of the formating is "1h 5m21s". On line 149, toTime() returns a wrong value if its formatted this way, it's working fine with "1h 5m 21s" (note the additional blank space between 'm' and '2'. I don't know if its safe to change the regex - what do you mean @fabiospampinato?
